service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateAksStep.java

```diff
@@ -185,6 +185,7 @@ private KubernetesCluster waitAndMaybeGetAksProvisioned(String mrgName, String a
         }
       } while (aksNotProvisioned);
     } catch (InterruptedException e) {
+      Thread.currentThread().interrupt();
```

Comment:
Was this just for debugging?

Reply:
We need to propagate interruption status. Once we handle this exception the flag is cleared, but we need to know upstream that we were interrupted.